src/main/java/core/org/akaza/openclinica/dao/hibernate/multitenant/MultiTenantFilter.java

```diff
@@ -73,6 +73,8 @@ public DataSource getDataSource() {
         if (session != null) {
             session.setAttribute(tenantKey, tenant);
         }
+        request.setAttribute("enableEmbeddedReports", CoreResources.getField("insight.enableEmbeddedReports"));
```

Comment:
rename the datainfo property to just "enableEmbeddedReports". We don't need the word insight before it as we want to keep this generic so that it would be easier to replace insight with some other reporting tool.

Reply:
Also, set the default value for this property to false in "datainfo.properties" file

Reply:
@svadla-oc roger
renamed and set to false